src/index.js

```diff
@@ -328,5 +346,87 @@ module.exports.create = function (opts) {
 };
 
 module.exports.version = function (cb) {
-    run(_command('version'), cb);
+    module.exports._run(_command('version'), cb);
+};
+
+module.exports.boxAdd = function (box, args, cb) {
+    if (typeof box !== 'string' && cb) {
+        cb('box must be provided as a string');
```

Comment:
Similar to the previous comment. I think the `return cb(` would better serve us here.